### PR TITLE
Improve chatcommand params consistency

### PR DESCRIPTION
**Pull request by Ezhh**
This makes params shown for commands when using "/help all" consistent. Suggestions are very welcome if it can be improved further.

### CODE REVIEW

**Comment by paramat** [comment]
<name>
means 'enter the 'name' variable'.
all
means enter the word 'all'
[ ]
means this is optional.
|
means 'OR'.
( )
defines the scope of the ORs.

File: builtin/game/chatcommands.lua [hunk 1]
```diff
@@ -468,7 +468,7 @@ local function emergeblocks_progress_update(ctx)
 end
 
 core.register_chatcommand("emergeblocks", {
-	params = "(here [radius]) | (<pos1> <pos2>)",
+	params = "(here [<radius>]) | ((<pos1>) (<pos2>))",
```
Comment (paramat) [inline builtin/game/chatcommands.lua]: This and the 2 commands below expect pos1 pos2 in the form
(x1, y1, z1) (x2, y2, z2)
I feel the double-bracketing here confuses showing the scope of OR with having to enter "(X,Y,Z)".
Better i ithink to remove the inner brackets and make the format clear in the description.

**Comment by paramat** [comment]
+1 if my line comment is addressed.

File: builtin/game/chatcommands.lua (hunk 1 above)
Comment (Ezhh) [inline builtin/game/chatcommands.lua]: Was thinking about this myself because those brackets annoyed me. Will adjust.

**Comment by SmallJoker** [comment]
Well done. You'll get my approval as well once the parenthesis above are fixed.

**Comment by Ezhh** [comment]
Updated :)
Thank you both.

**Comment by paramat** [comment]
:+1:

**Pull request merged**